Source/Atomic/Engine/EngineConfig.cpp

```diff
@@ -190,6 +190,14 @@ bool EngineConfig::LoadWindowConfig(const JSONValue& jwindow)
             engineConfig_["Borderless"] = GetBoolValue(jvalue, false);
         else if (key == "resizable")
             engineConfig_["WindowResizable"] = GetBoolValue(jvalue, false);
+        else if (key == "width")
+            engineConfig_["WindowWidth"] = GetIntValue(jvalue, false);
+        else if (key == "height")
+            engineConfig_["WindowHeight"] = GetIntValue(jvalue, false);
+        else if (key == "positionx")
+            engineConfig_["WindowPositionX"] = GetIntValue(jvalue, false);
+        else if (key == "positiony")
+            engineConfig_["WindowPositionY"] = GetIntValue(jvalue, false);
```

Comment:
Yeah we really need to configure starting location and size externally :) Looking at the config/settings being read from the commandline between player, engine and embedded player it really looks like some kind of shared code might be good there as well.